RPS Project/Tests.py

```diff
@@ -0,0 +1,73 @@
+import random
+
+#sets paramaters, including the probability that of the "player's" choices, and starts the game
+def test(number, prob_0, prob_1):
+    global score, history, count, rock, paper
+    score = [0,0,0]
+    history = [0,0,0]
+    count = [0]
+    rock = prob_0
+    paper = prob_1
+    game(number)
+
+#Returns game result
+def judge(sign_1, sign_2):
```

Comment:
You should be using your tests to test `game.py`, not a copy of those functions. Import the functions you want to test.